src/related-components/RelatedCard.js

```diff
@@ -10,7 +10,10 @@ import CardActions from "@material-ui/core/CardActions";
 import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
 import withStyles from "@material-ui/core/styles/withStyles";
-
+import Grid from "@material-ui/core/Grid";
```

Comment:
You can import the Typography, Container, IconButton, Card, CardActions, CardContent, and CardMedia together if you want to. Since they all form "@material-ui/core"